test/integration/gcp/README.md

```diff
@@ -0,0 +1,66 @@
+# Example InSpec Profile For GCP
```

Comment:
This README doesn't seem to be specific to the forseti project, can we remove or or update it to be more relevant to the forseti test coverage?